docs/tutorials/ad-hoc-simulation/planning-simulation.md

```diff
@@ -4,11 +4,11 @@
 
 Download and unpack a sample map.
 
-- Click [here](https://drive.google.com/file/d/1499_nsbUbIeturZaDj7jhUownh5fvXHd/view?usp=sharing) to download.
-- Unpack it by running the following command.
+- You can also download [the map](https://drive.google.com/file/d/1499_nsbUbIeturZaDj7jhUownh5fvXHd/view?usp=sharing) manually.
 
 ```bash
-unzip -d ~/Downloads/ ~/Downloads/sample-map-planning.zip
+wget -O ~/autoware/sample-map-planning.zip 'https://docs.google.com/uc?export=download&id=1499_nsbUbIeturZaDj7jhUownh5fvXHd'
```

Comment:
I think we should avoid using `~/autoware` because it's not good to put data in Autoware's workspace. We sometimes delete the directory, then we have to re-download it.
@evshary What do you think?

```suggestion
mkdir ~/map
wget -O ~/map/sample-map-planning.zip 'https://docs.google.com/uc?export=download&id=1499_nsbUbIeturZaDj7jhUownh5fvXHd'
```

Reply:
I agree with this. This can keep "autoware" folder clean.
But I have one question about this:
If we put data under `~/map` and also mount the `~/map` folder from the host into the container,
then we need to create the `~/map` folder before using docker.
Should this step be mentioned in [docker installation](https://autowarefoundation.github.io/autoware-documentation/main/installation/autoware/docker-installation/#how-to-set-up-a-development-environment)?

I think it'll be great if users can try these Autoware examples with a simple copy and paste.

Reply:
I added `autoware_map` folder for the map data now.